doc/source/whatsnew/v1.1.2.rst

```diff
@@ -23,6 +23,8 @@ Fixed regressions
 - Regression in :meth:`DataFrame.replace` where a ``TypeError`` would be raised when attempting to replace elements of type :class:`Interval` (:issue:`35931`)
 - Fix regression in pickle roundtrip of the ``closed`` attribute of :class:`IntervalIndex` (:issue:`35658`)
 - Fixed regression in :meth:`DataFrameGroupBy.agg` where a ``ValueError: buffer source array is read-only`` would be raised when the underlying array is read-only (:issue:`36014`)
+- Fixed regression in :meth:`Series.__getitem__` incorrectly raising when the input was a tuple. (:issue:`35534`)
```

Comment:
also don't normally add the full stop on the release notes.

Reply:
@simonjayhawkins Not sure what you mean by "full stop" here.

Reply:
period